lib/rubocop/cop/method_complexity_patch.rb

```diff
@@ -0,0 +1,10 @@
+# Monkey patching! Here be dragons.
+module RuboCop
+  module Cop
+    module MethodComplexity
+      def add_offense(node, loc, message = nil, severity = nil)
```

Comment:
FWIW: I'm a big fan of good commit messages and a team that goes to `git blame` immediately over leaving "wtf?" comments in the source. In this case a small comment and big commit message seems like an OK compromise though.